superset-frontend/src/SqlLab/components/QuerySearch.jsx

```diff
@@ -278,9 +287,9 @@ class QuerySearch extends React.PureComponent {
           {this.state.queriesLoading ? (
             <Loading />
           ) : (
-            <div
+            <TableStyles
               className="scrollbar-content"
-              style={{ height: this.props.height }}
+              height={this.props.height}
```

Comment:
This line shouldn't be needed, as it's covered in the `styled` implementation. But just to reiterate, I'd love to not need this prop at all :D

Reply:
Removed with the refactor